circuit-lib/circuitlib-rs/parse_vk_to_rust.js

```diff
@@ -0,0 +1,133 @@
+var ffjavascript = require('ffjavascript');
+const {unstringifyBigInts, leInt2Buff} = ffjavascript.utils;
+var fs = require("fs");
+const process = require('process');
```

Comment:
we have this function in zk.js and an example of using it in circuit-lib.circom

Reply:
zk.js is broken, so I moved the script to circom-lib.circom/scripts.